examples/src/main/java/io/dapr/examples/configuration/grpc/README.md

```diff
@@ -0,0 +1,92 @@
+## Retrieve Configurations via Configuration API
+
+This example provides the different capabilities provided by Dapr Java SDK for Configuration. For further information about Configuration APIs please refer to [this link](https://docs.dapr.io/developing-applications/building-blocks/configuration/)
+**This API is available in Preview Mode**.
+
+## Pre-requisites
+
+* [Dapr and Dapr Cli](https://docs.dapr.io/getting-started/install-dapr/).
+* Java JDK 11 (or greater): [Oracle JDK](https://www.oracle.com/technetwork/java/javase/downloads/index.html#JDK11) or [OpenJDK](https://jdk.java.net/13/).
+* [Apache Maven](https://maven.apache.org/install.html) version 3.x.
+
+### Checking out the code
+
+Clone this repository:
+
+```sh
+git clone https://github.com/dapr/java-sdk.git
+cd java-sdk
+```
+
+Then build the Maven project:
+
+```sh
+# make sure you are in the `java-sdk` directory.
+mvn install
+```
+
+Get into the examples' directory:
+```sh
+cd examples
+```
+
+### Using the ConfigurationAPI
+
+The java SDK exposes several methods for this -
+* `client.getConfiguration(...)` for getting a configuration for a single key.
+* `client.getConfigurations(...)` for getting a configurations by passing a variable no. of arguments or list of argumets.
+* `client.getAllConfigurations(...)` for getting all configurations.
+* `client.subscribeToConfigurations(...)` for subscribing to a list of keys for any change.
+
+### Running the example
+The code uses the `DaprPreviewClient` created by the `DaprPreviewClientBuilder`.
+
+```bash

```

Comment:
how are the initial configuration items being populated for this example?

Reply:
Updated the command needed for populating the initial data